assets/app/views/projects.html

```diff
@@ -1,28 +1,94 @@
+    </div>
+    <div style="margin-top: 10px;">
+      A project admin can add you as an admin to a project by running the command
+      <code>oc policy add-role-to-user admin {{user.metadata.name || '&lt;YourUsername&gt;'}} -n &lt;projectname&gt;</code>
     </div>
   </div>
-  <div ng-if="emptyMessage && (projects | hashSize) == 0">{{emptyMessage}}</div>
-  <div ng-if="canCreate === false" style="margin-top: 10px;">
-    <span ng-if="!newProjectMessage">A cluster admin can create a project for you by running the command
-      <code>oadm new-project &lt;projectname&gt; --admin={{user.metadata.name || '&lt;YourUsername&gt;'}}</code></span>
-    <span ng-if="newProjectMessage" ng-bind-html="newProjectMessage | linky" style="white-space:pre;"></span>
-  </div>
-  <div style="margin-top: 10px;">
-    A project admin can add you as an admin to a project by running the command
-    <code>oc policy add-role-to-user admin {{user.metadata.name || '&lt;YourUsername&gt;'}} -n &lt;projectname&gt;</code>
+  <div ng-if="showGetStarted">
+    <h1>Welcome to OpenShift.</h1>
+
+    <p class="text-muted">OpenShift helps you quickly develop, host, and scale applications.</p>
+
+    <div ng-if="canCreate" class="empty-state-msg">
+      <p class="lead">Let's get started.</p>
+
+      <ol>
+        <li>
+          <strong>Create a project.</strong>
+          <div class="help">Organize your content using projects.</div>
+        </li>
+
+        <li>
+          <strong>Add components.</strong>
+          <div class="help">Build your application by adding</div>
+
+          <ul>
+            <li>
+              <strong>Source Code</strong> &mdash;
+
+              Create applications from code in a Git repository.
+
+              <span class="hidden-xs">
+                OpenShift builds your code into an application image and
+                deploys it behind a load-balanced service.
+              </span>
+            </li>
+            <li>
+              <strong>Templates</strong> &mdash;
+
+              Add components like databases or QuickStart applications to your
+              projects using templates.
+
+              <span class="hidden-xs">
+                QuickStart applications give you a starting point for
+                creating an application in your favorite language.
+              </span>
+            </li>
+          </ul>
+        </li>
+      </ol>
+
+      <p class="hidden-xs">
+        Once your application is running, you can configure triggers to
+        update your application when things change, set up deployment
+        strategies like rolling deployments, and scale your application as
+        needed.
+      </p>
+
+      <a href="/createProject" class="lead">
+        Create your first project
+      </a>
+    </div>
+
+    <div ng-if="!canCreate" class="empty-state-msg">
+      <p class="lead">Unfortunately, you don't have authority to create projects.</p>
```

Comment:
remember, `newProjectMessage` can contain the customized message from the admin. They might have enabled a way for users to create projects for themselves, so we have to show it if it exists (using `linky`), and we shouldn't pair it with an "Unfortunately..." header